client/src/components/QA/QACardAnswers.jsx

```diff
@@ -13,6 +13,8 @@ const QACardAnswers = ({ currentAnswers }) => {
   const [answersToShowLength, setAnswersToShowLength] = useState(2);
   // toggle is to make sure we avoid infinite rerender loop
   const [toggle, setToggle] = useState(false);
+  const [helpfulArray, setHelpfulArray] = useState([]);
```

Comment:
Super ultra turbo nitpick, I would change the name of this to something more semantically relevant, like `answersMarkedHelpful`. The array part is clear enough with the `useState` call